src/templater.coffee

```diff
@@ -0,0 +1,39 @@
+JSCK = require "jsck"
+{async, read, merge } = require "fairmont"
+{yaml} = require "panda-serialize"
+pandatemplate = require "panda-template"
+
+module.exports = class Templater
+  @read: async (templatePath, schemaPath) ->
+    template = yield read templatePath
+    if schemaPath
+      schema = yaml yield read schemaPath
+    else
+      schema =
+        $schema: "http://json-schema.org/draft-04/schema#"
+        type: "object"
+        minProperties: 1
+
+    new @ template, schema
+
+  constructor: (@template, @schema) ->
+    @validator = new JSCK.draft4 @schema
+
+  render: (config) ->
+    {valid, errors} = @validator.validate config
```

Comment:
Shouldn't we have a `validate` method? Don't we sometimes want to do a simple validation first?